tests/test_plugins_general_plugins.py

```diff
@@ -60,8 +60,8 @@ def test_plugin_description_in_dict_multiple():
     ]
 
 
-@patch.object(plugin.plugin, "load")
+@patch.object(plugin, "load")
 @patch.object(plugin, "get_exported_plugins")
-def test_categorize_plugins(mocked_export, mocked_load):
+def _test_categorize_plugins(mocked_export, mocked_load):
```

Comment:
Disabled test?

Reply:
That file should not have been altered, reset.